src/datatable/math.py

```diff
@@ -72,9 +71,33 @@
     trunc,
 )
 
+from datatable.lib._datatable import (
+    isna
+)
+
+from datatable.lib import core
+import math
+
+__all__ = [
+    "isna"
+]
+
 e = 2.718281828459045
 golden = 1.618033988749895
 pi = 3.141592653589793
 tau = 6.283185307179586
 nan = float('nan')
 inf = float('inf')
+
+
+def isna(arg):   
+    if (not isinstance(arg, dict)
+        and (isinstance(arg, core.FExpr)
+        or (arg and (hasattr(arg, "__getitem__") and isinstance(arg[0], core.FExpr))))):
+        return core.isna(arg)
+    if isinstance(arg, dict) and isinstance([*arg.values()][0], core.FExpr):
+        return core.isna(arg)
+    try:
+        return math.isnan(arg)
+    except TypeError:
+        return arg is None
```

Comment:
Do you want to do here something similar we are doing for `sum`, `min` and `max`?

Reply:
It is based on the `sum` implementation. Am I missing a condition?

Reply:
We have been doing that for sum, because there is a built-in python sum function that we didn’t want to spoil if someone imports dt’s sum. For isna there is no python’s isna.

Reply:
Yeah, no one will install datatable for the sole purpose of using `dt.isna()`, when there is a standard `math.isnan()`.